src/sensors/odometer/interrupt/DirectionlessOdometer.cpp

```diff
@@ -15,8 +15,8 @@ DirectionlessOdometer::DirectionlessOdometer(Runtime& runtime,
                                              uint8_t pulsePin,
                                              InterruptCallback callback,
                                              unsigned long pulsesPerMeter)
-    : mPulsesPerMeterRatio{ pulsesPerMeter > 0 ? static_cast<float>(pulsesPerMeter) / 100.0F
-                                               : kDefaultPulsesPerMeter / 100.0F }
+    : mPulsesPerMeterRatio{ pulsesPerMeter > 0 ? static_cast<float>(pulsesPerMeter) / 100.0f
+                                               : kDefaultPulsesPerMeter / 100.0f }
```

Comment:
:warning: **readability-uppercase-literal-suffix** :warning:
floating point literal has suffix 'f', which is not uppercase
```suggestion
                                               : kDefaultPulsesPerMeter / F```